src/main.js

```diff
@@ -336,11 +336,11 @@ class MiniGraphCard extends LitElement {
       </div>` : '';
   }
 
-  generateLegend(index) {
+  computeLegend(index) {
     let legend = this.computeName(index);
     const state = this.getEntityState(index);
 
-    const show_legend_state = this.config.entities[index].show_legend_state ?? false;
+    const { show_legend_state = false } = this.config.entities[index];
```

Comment:
We have already discussed this earlier:
the `show_legend_state` is an option inside `config.entities`.
What is a purpose of assigning a bool value to an OBJECT?
Why not assigning it to `this.config.entities[index].show_legend_state` instead?

Reply:
See @akloeckner review that I have unresolved.
Personally, both are fine for me.

Reply:
It's just a shorthand for the other version. 

This is equivalent: 
```js
a = config.a;
{a} = config;
```

So, actually, it's a matter of style. It has been used before in the project. That's why I suggested to use it again. 

Actually, the default value isn't used so far. But I think it cannot harm.

Reply:
I have a very little experience in JS, so asking silly questions.
I believed that "this.config.entities[index]" is a particular member of "entities" list - i.e. it is an object containing all options inside a particular entity (including that "show_legend_state" option). But we need to take a particular "show_legend_state" option - that is why I believed that we need to write
```
const { show_legend_state = false } = this.config.entities[index].show_legend_state;
```
If I am wrong - let's finalize this PR then:
- generate js file for testing,
- test it together,
- then create a Release if everything is OK.

Reply:
You're right about the structure of the config object. The "destructuring assignment" is just a short way to extract properties of the object into single variables. 

See the documentation here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment